lib/sentry.server.js

```diff
@@ -0,0 +1,20 @@
+import * as Sentry from '@sentry/node'
+import { Dedupe, ExtraErrorData, RewriteFrames, Transaction } from '@sentry/integrations'
+
+export default function (ctx, inject) {
+  const opts = Object.assign({}, <%= serialize(options) %>, {
+    integrations: [
+        new Dedupe(),
+        new ExtraErrorData(),
+        new RewriteFrames(),
+        new Transaction()
+      ]
+  })
+
+  if (!opts.disabled) {
```

Comment:
Both in .server and .client, this check is no longer necessary because module will exit early if `disabled` is true.

Reply:
@rchl its different check
disableClientSide vs config.disabled
 e.g. someone want to disable logging while leaving this.$sentry working in code as opposed to completely disabling. Before it was the same, so i just left logic as it is. Correct me if i wrong somewhere